src/Deck.java

```diff
@@ -0,0 +1,75 @@
+public class Deck {
+    private String[] words = new String[100];
+    private String[] definitions = new String[100];
+    private int numWords = 0;
+    private int numDefinitions = 0;
+
+    /**
+     * Takes the words from the text file and saves them to an array. 
+     * @param q
+     */
+    public void addWord(String q) {
```

Comment:
you should change the parameter names to be more meaningful. You can also add a description of them in the documentation comment, for example

```java
/**
 * Takes the word from the text file and saves them to an array.
 * @param word The word to be saved
 */
```

which looks quite nice when viewed using VS Code's documentation hovers